JMMServer/Providers/MyAnimeList/MALHelper.cs

```diff
@@ -530,15 +532,21 @@ public static void UpdateMALSeries(AnimeSeries ser)
                             ser.GetAnime().AnimeID);
                         continue;
                     }
+
+                    string confirmationMessage = "";
+                    if (animeListHashtable.ContainsKey(malID))
```

Comment:
> I am not sure we can rely on this as MAL list can be cached, there is an open issue on their forums about anime list not always returning latest data.

I see what you mean. With the animeListHashTable, it is only populated if we have Never Decrease Series Status set to True. If it is false, then the Upload function will just default to calling "Add" for every series, which is not ideal as that is the current broken behavior. If it is true, then everything works fine unless mysteriously on the MAL end the anime list is not returning latest data which is out of our hands.